packages/client-api-schema/src/types.ts

```diff
@@ -167,28 +162,43 @@ export type ChallengeChannelRequest = JsonRpcRequest<'ChallengeChannel', {channe
 export type ChallengeChannelResponse = JsonRpcResponse<ChannelResult>;
 
 // Budget
+
+export interface ChannelBudget {
+  channelId: string;
+  status: 'pending' | 'available';
+  amount: string;
+}
+
 export interface TokenBudget {
   token: string;
-  pending: Balance;
-  free: Balance;
-  inUse: Balance;
-  direct: Balance;
+  availableReceiveCapacity: string;
+  availableSendCapacity: string;
+  channels: ChannelBudget[];
```

Comment:
I switched this from a `Record` as specified in the notion doc to keep it consistent with `budgets`. I think we should either using a `Record` or `Array` consistently in the API types but I'm ok using either depending on what's easiest for `web3-torrent`.